pkg/configresolver/config_resolver_test.go

```diff
@@ -0,0 +1,64 @@
+package configresolver
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/mitchellh/go-homedir"
+	"github.com/stretchr/testify/assert"
+	"golang.stackrox.io/kube-linter/pkg/config"
+)
+
+var cfg *config.Config = new(config.Config)
+
+func TestHomeIgnorePath(t *testing.T) {
```

Comment:
How about using table test for this tests?

Reply:
good idea [90986c0](https://github.com/stackrox/kube-linter/pull/353/commits/90986c0d61305c473b671a7e9d26c7af7251503b)